meson.build

```diff
@@ -0,0 +1,66 @@
+project('SLB', 'c',
+  version : '0.1',
+  default_options : ['warning_level=3'])
+
+cc = meson.get_compiler('c')
+m_dep = cc.find_library('m', required : false)
+
+url = 'https://raw.githubusercontent.com/Cyan4973/xxHash/dev'
+run_command('wget', '-nv', '-N', url+'/xxhash.h')
+run_command('wget', '-nv', '-N', url+'/xxhash.c')
```

Comment:
Please use https://mesonbuild.com/Wrap-dependency-system-manual.html instead.

Reply:
ah cool, xxhash is available https://mesonbuild.com/Wrapdb-projects.html

I see it there. cool